extensions/cornerstone-dicom-seg/src/getToolbarModule.ts

```diff
@@ -33,6 +65,21 @@ export function getToolbarModule({ servicesManager }: withAppTypes) {
           };
         }
 
+        const activeRepresentations = segmentationService.getSegmentationRepresentations(
+          viewportId,
+          {
+            segmentationId: activeSegmentation.segmentationId,
+            type: segmentationRepresentationType,
+          }
+        );
+        if (!activeRepresentations?.length) {
+          return {
+            disabled: true,
+            disabledText: `Active segmentation is not a ${segmentationRepresentationType} segmentation`,
+            visible: customizationService.getCustomization('panelSegmentation.isMultiTab'),
```

Comment:
This little trick allows us to show ALL the tools for a particular representation type disabled (and not invisible) when the active seg is not of the specified representation type in multi-tab mode.